UI/Pages/AcademicMentors.cshtml

```diff
@@ -80,4 +80,8 @@
 		</ul>
 		</p>
 	</div>
+	<div class="govuk-grid-column-one-third">
+		<h2 class="govuk-heading-m">Find out how much funding you’ll get</h2>
```

Comment:
I think this needs to be lower on the page the same level as the title  "Employ your own tutor" under the home link. As in the wire frame

Reply:
In the wire frame they included the print page image at the top that's why they pushed it down Stephen and cross checked with Rory and he said he is happy with this as it will be consistency around the all pages like home page